commands

```diff
@@ -30,9 +30,16 @@ case "$1" in
     mkdir -p "$SERVICE_ROOT/config" || dokku_log_fail "Unable to create service config directory"
     echo -e "[mysqld]\nperformance_schema = 0" > "$SERVICE_ROOT/config/disable_performance_schema.cnf"
     rootpassword=$(openssl rand -hex 8)
-    password=$(openssl rand -hex 8)
     echo "$rootpassword" > "$SERVICE_ROOT/ROOTPASSWORD"
```

Comment:
We actually use the root user for mysql. That said, only a few of us use it so we could change it pretty easily.